src/tree/DatabaseAccountOptionsStep.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { IDBAWizardContext } from './IDBAWizardContext';
+
+export class DatabaseAccountOptionsStep extends AzureWizardPromptStep<IDBAWizardContext> {
```

Comment:
Remove this file and `IDBAWizardContext` in this PR since we're handling entry points separately